src/include/denc.h

```diff
@@ -44,11 +44,12 @@
 #include "buffer.h"
 #include "byteorder.h"
 
-template<typename T, typename VVV=void>
+template<typename T, typename=void>
 struct denc_traits {
   static constexpr bool supported = false;
   static constexpr bool featured = false;
   static constexpr bool bounded = false;
+  static constexpr bool with_dencv = false;
```

Comment:
a short explanation is in the commit message. and i thought it was short. but yeah, we'd better use a expressive name. will do.